README.md

```diff
@@ -145,7 +146,7 @@ pip install dist/*
 ```
 2. UT
 ``` shell
-sh test_python_ut.sh
```

Comment:
The docs/install/install.md also should be modified.

Reply:
modified